x-pack/plugins/alerting/public/plugin.ts

```diff
@@ -68,6 +69,7 @@ export interface AlertingPluginStart {
   spaces: SpacesPluginStart;
   unifiedSearch: UnifiedSearchPublicPluginStart;
   data: DataPublicPluginStart;
+  serverless?: ServerlessPluginStart;
```

Comment:
Is changing this type enough? How does this guarantee that `useKibana` has it whenever the hook is called?

Reply:
I think it correctly reflects (with the `?`) the fact that the `serverless` plugin is not required but optional in `alerting`'s `kibana.jsonc` config file